core/src/scss/components/masthead/_masthead.scss

```diff
@@ -0,0 +1,125 @@
+@charset "UTF-8";
+
+//
+// Masthead
+//
+// Masthead includes the Stanford brand bar (or a top stroke), skip link, logo lockup, site search and main navigation.
+//
+// .su-masthead--center - Center-aligned main nav
+// .su-masthead--right - Right-aligned main nav
+// .su-masthead--light - Masthead with the light variant of main nav
+// .su-masthead--dark - Masthead with a dark color theme
+//
+// Markup: ../templates/components/masthead/masthead.twig
+//
+// Style guide: Components.Masthead
+//
+
+.su-masthead {
+  @include box-shadow($depth: 'shallow', $stroke: 0);
+  position: relative;
+  background-color: color(bg);
+  max-width: 100%;
+
+  > section:last-of-type {
+    @include flex-container;
+    @include centered-column;
+    @include modular-spacing('padding', 1 null null);
+  }
+
+  .su-lockup {
+    @include modular-spacing('margin', null null 0);
+    max-width: fit-content;
+
+    @include grid-media-max('md') {
+      @include flex-column($columns: 10);
```

Comment:
No default allowed in calling a mixin. 

`@include flex-column($columns:);`
or
`@include flex-column(10);`

Reply:
Good to know. All fixed! 😄